Makefile

```diff
@@ -568,6 +574,7 @@ copy-files:
 	-chown salt:salt $(DESTDIR)/srv/salt/ceph/configuration/files/ceph.conf.checksum || true
 
 install: copy-files
+	sed -i "s/DEVVERSION/"$(VERSION)"/" setup.py
```

Comment:
This will add unstaged changes to the local git repo

Reply:
Right my bad. Suggestion: use a ```sed "s/DEVVERSION/"$(VERSION)"/" setup.py.in > setup.py``` to replace the version and (git)ignore setup.py?